crmsh/bootstrap.py

```diff
@@ -2335,10 +2334,14 @@ def bootstrap_join(context):
             _context.cluster_node = cluster_node
 
         join_ssh(cluster_node)
-        join_remote_auth(cluster_node)
-        join_csync2(cluster_node)
-        join_ssh_merge(cluster_node)
-        join_cluster(cluster_node)
+
+        lock_inst = join_lock.JoinLock(cluster_node)
+        with lock_inst.lock():
+            setup_passwordless_with_other_nodes(cluster_node)
```

Comment:
Consider on this time other node was joining, we should call `setup_passwordless_with_other_nodes` here to make sure init node already has latest membership